content/en/docs/examples/virtual-machines/autoregistration/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Automated Workload Entry Creation [experimental]
+description: Learn how to use the experimental automated VM registration feature.
+  Istio mesh.
+weight: 70
+keywords:
+- kubernetes
+- vms
+- virtual-machines
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{< /warning >}}
+
+Istio 1.8 introduces a new configuration resource, [Workload Group](/docs/reference/config/networking/workload-group/), that can be used to automate
+the creation of [Workload Entries](/docs/reference/config/networking/workload-entry/).
+This example shows how to use a `WorkloadGroup` to integrate a virtual machine, or a bare metal host into a
+single network Istio mesh deployed on Kubernetes without manual `WorkloadEntry` creation. This approach requires L3 connectivity
+between the virtual machine, and the Kubernetes cluster.
+
+## Prerequisites
+
+- One or more Kubernetes clusters with versions: {{< supported_kubernetes_versions >}}.
+
+- Virtual machines must have L3 IP connectivity to the endpoints in the mesh.
+  This typically requires a VPC or a VPN, as well as a container network that
+  provides direct (without NAT or firewall deny) routing to the endpoints. The
+  machine is not required to have access to the cluster IP addresses assigned by
+  Kubernetes.
+
+- Installation must be completed using [virtual machine installation](/docs/setup/install/virtual-machine) instructions.
+
+## Prepare the guide environment
+
+Set the environment variables `VM_NAMESPACE` and `SERVICE_ACCOUNT` (use the same values that you used during installation):
+
+{{< text bash >}}
+$ VM_NAMESPACE="<the name of your service namespace>"
+$ SERVICE_ACCOUNT="<name of the Kubernetes service account you want to use for your VM>"
+{{< /text >}}
+
+### Running services on the virtual machine
+
+1. Setup an HTTP server on the virtual machine to serve HTTP traffic on port 8080:
+
+    {{< text bash >}}
+    $ python -m SimpleHTTPServer 8080
+    {{< /text >}}
+
+    {{< warning >}}
+    You may have to open firewalls to be able to access the 8080 port on your virtual machine
+    {{< /warning >}}
+
+1. Add an associated Service to the mesh:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl -n "${VM_NAMESPACE}" apply -f -
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: auto-cloud-vm
+      labels:
+        app: auto-cloud-vm
+    spec:
+      ports:
+      - port: 8080
+        name: http
+        targetPort: 8080
+      selector:
+        app: auto-cloud-vm
+    EOF
+    {{< /text >}}
+
+## Configure VM for Auto-Registration
+
+1. Create the auto-registration group.
+
+    `WorkloadGroup`provides a template to automatically create a `WorkloadEntry`for each connected VM instance.
```

Comment:
FIx here and several other places: https://github.com/istio/istio.io/pull/8372#issuecomment-718949713
```suggestion
    `WorkloadGroup`provides a template to automatically create a `WorkloadEntry` for each connected VM instance.
```